p62/mini-aes/src/lib.rs

```diff
@@ -1,14 +1,49 @@
-pub fn add(left: u64, right: u64) -> u64 {
-    left + right
+use miniaes_sys as sys;
+use std::mem::MaybeUninit;
+
+pub struct AESWrapper {
+    keys: sys::RoundKeys,
 }
 
-#[cfg(test)]
-mod tests {
-    use super::*;
+impl Default for AESWrapper {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl AESWrapper {
+    pub fn new() -> Self {
+        Self {
+            keys: unsafe { MaybeUninit::zeroed().assume_init() },
```

Comment:
This method should have the following signature:
```rust
pub fn new(key: &[u8; 16]) -> Self { ... }
```
And you can remove the `from_key` method.